### PR TITLE
VC-4222 add graph ql tables

### PR DESCRIPTION
**Pull request by alex-bodnar**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: hasura/metadata/actions.yaml [hunk 1]
```diff
@@ -1,291 +1,127 @@
...
-    handler: http://localhost:3000/delegation_reward
-    output_type: "[ActionDelegationReward]"
-    arguments:
-    - name: address
-      type: String!
-    - name: height
-      type: Int
-    type: query
+    kind: ""
+    handler: http://localhost:3000/delegation
```
Comment (DirusK) [inline hasura/metadata/actions.yaml:14]: можно вынести в отдельную переменную хост  
допустим если в дальнейшем потребуется переключиться на дев мб или ещё куда, было бы удобно менять это в одном месте, а не по всему ямлу

Comment (alex-bodnar) [reply hasura/metadata/actions.yaml:14]: Это, CLI генерила

**Comment by DirusK** [comment]
LGTM

**Review by IamStubborN — APPROVED** [review]
LGTM

**Pull request merged**